sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ManagementChannel.java

```diff
@@ -59,12 +60,21 @@ public Mono<AmqpAnnotatedMessage> send(AmqpAnnotatedMessage message) {
             return channel.sendWithAck(protonJMessage)
                 .handle((Message responseMessage, SynchronousSink<AmqpAnnotatedMessage> sink) ->
                     handleResponse(responseMessage, sink, channel.getErrorContext()))
-                .switchIfEmpty(Mono.error(() -> new AmqpException(true, String.format(
-                    "entityPath[%s] No response received from management channel.", entityPath),
-                    channel.getErrorContext())));
+                .switchIfEmpty(errorIfEmpty(channel, null));
         }));
     }
 
+    private <T> Mono<T> errorIfEmpty(RequestResponseChannel channel, com.azure.core.amqp.models.DeliveryState deliveryState) {
```

Comment:
nit: keeping methods that are the same visibility together. There are private methods below.